shell/platform/embedder/embedder.h

```diff
@@ -611,6 +611,36 @@ typedef struct {
   int64_t buttons;
 } FlutterPointerEvent;
 
+typedef enum {
+  kFlutterKeyEventKindUp = 1,
+  kFlutterKeyEventKindDown,
+  kFlutterKeyEventKindRepeat,
+} FlutterKeyEventKind;
+
+typedef struct {
+  /// The size of this struct. Must be sizeof(FlutterKeyEvent).
+  size_t struct_size;
+  // Timestamp in microseconds.
+  double timestamp;
+  // The event kind.
+  FlutterKeyEventKind kind;
+  // The physical key of the event, distinguished by the HID code.
+  uint64_t physical;
+  // The logical key of the event, usually the effect of the key without
+  // modifier, but might include modifier if the information is not available.
+  // Can be 0 for empty.
```

Comment:
It's not obvious what this means exactly; what is the definitive source of numbers for logical keys?

Since this API surface is intended for use by third-party developers as well, it's important not to assume too much context in the comments.